python/res/fm/ecl/ecl_run.py

```diff
@@ -238,6 +183,7 @@ def initRedirect(self):
 
     def execSerialEclipse(self):
         os.execv( self.sim.executable , [ self.sim.executable , self.base_name ])
+        irint("cwd: {}  calling {}  {}".format(os.getcwd(), self.sim.executable, self.base_name))
```

Comment:
Either this is never tested or the tests will fail...

Reply:
It might actually "work" - because due to the `execv()` call the line should never be executed - test or not. Have removed it.

Reply:
OK